dashboard/test/fixtures/test-script-course-version.script

```diff
@@ -0,0 +1,3 @@
+family_name 'xyz'
+version_year '1234'
+is_course true
```

Comment:
Is there any benefit to having some lesson groups, lessons and levels in here if this is meant to be used for an integration test?

Reply:
I'm not sure I see it, even though I'm calling it an "integration test" for lack of better terms, I want to test something relatively specific around seeding and creating CourseVersions. I don't think lesson groups / lessons / levels should affect that. Were you thinking of anything in particular?

Reply:
No, I didn't have anything specific was just wondering